ivy/functional/ivy/image.py

```diff
@@ -87,6 +89,7 @@ def gradient_image(x):
 
     Examples
```

Comment:
great work so far, but please add more examples by following the points stated here (https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html)